aws_xray_sdk/core/async_context.py

```diff
@@ -1,7 +1,10 @@
 import asyncio
+import sys
 
 from .context import Context as _Context
 
+_GTE_PY37 = sys.version_info.minor >= 7
+
```

Comment:
I would also add condition which checks sys.version_info.major is 2 and 3 because in the case of major version 4.0 above condition would break.

Reply:
Do we need a guard against 4? I think we do need to make sure its python3. I forgot python 2.7 had asyncio support. So I assumed no python2 code would call this code and if it did it would error for other reasons.

So basically just this.
```python
_GTE_PY37 = sys.version_info.major == 3 and  sys.version_info.minor >= 7
```

Reply:
Yep. I think this makes sense.